spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -39,6 +39,8 @@
     )
 
     expect(response.valid?(original_challenge, original_origin)).to eq(true)
+    expect(response.attestation_type).to eq(WebAuthn::AttestationStatement::Base::ATTESTATION_TYPE_BASIC_OR_ATTCA)
+    expect(response.attestation_trust_path).to be_kind_of(OpenSSL::X509::Certificate)
```

Comment:
From what I understand W3C recommendations states it should return `x5c` which stands for a 1 element array, not the actual element.
https://www.w3.org/TR/2018/CR-webauthn-20180807/#fido-u2f-attestation